show/muxcable.py

```diff
@@ -2255,4 +2271,190 @@ def muxdirection(db, port, json_output):
         if rc_exit == False:
             sys.exit(EXIT_FAIL)
 
+@muxcable.command()
+@click.argument('port', metavar='<port_name>', required=True, default=None)
+@click.argument('option', required=False, default=None)
+@click.option('--json', 'json_output', required=False, is_flag=True, type=click.BOOL, help="display the output in json format")
+@clicommon.pass_db
+def queueinfo(db, port, option, json_output):
+    """Show muxcable queue info information, preagreed by vendors"""
+
+    port = platform_sfputil_helper.get_interface_name(port, db)
+    delete_all_keys_in_db_tables_helper()
+
+    if port is not None:
+
+        res_dict = {}
+        result = {}
+        param_dict = {}
+        param_dict["option"] = option
 
+
+        res_dict[0] = CONFIG_FAIL
+        res_dict[1] = "unknown"
+
+        res_dict = update_and_get_response_for_xcvr_cmd(
+            "get_ber", "status", "True", "XCVRD_GET_BER_CMD", "XCVRD_GET_BER_CMD_ARG", "XCVRD_GET_BER_RSP", port, 100, param_dict, "queue_info")
+
+        if res_dict[1] == "True":
+            result = get_result(port, res_dict, "fec" , result, "XCVRD_GET_BER_RES")
+
+
+        delete_all_keys_in_db_tables_helper()
```

Comment:
can we add it as decorator? do we need in destructor call? I don't see this needed here after command response is completed. if ycabled response failed...I am afraid if you are clearing this for commands from other CLI/shell context

Reply:
fixed, called in subroutine now